feature-libs/product/future-stock/styles/_future-stock-accordion.scss

```diff
@@ -21,6 +21,15 @@ cx-future-stock-accordion {
     cx-icon {
       padding-inline-start: 0.375rem;
     }
+
+    @include forFeature('a11yVisibleFocusOverflows') {
```

Comment:
I think I've have fixed this one too here: https://github.com/SAP/spartacus/pull/19579/files.
`a11yVisibleFocusOverflows` has been enabled recently since it's 6 moths old. We probably should not add to it anymore. Do you think we can go with the solution I've linked?

Reply:
Nice catch! Thanks for pointing that out - I'll switch to the `a11yCroppedFocusRing`